src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3309,18 +3439,42 @@ export class StudyViewPageStore {
             return '';
     }
 
+    public async getFusionGenesDownloadData() {
```

Comment:
I don't think this needs to be async as it doesn't seem to do any fetching/awaiting.  This function I think is really worthy of a test.

Reply:
agree. no need for async here.